test/behave/12_tracker.feature

```diff
@@ -4,3 +4,14 @@ Feature: tracker
     Given an English speaking user
     When the user says "give me location of the mark"
 	  Then mycroft reply should contain "home"
+
+  Scenario Outline: where skill conflict>
```

Comment:
Could use something more descriptive like: 
> "Intent conflict when asking where something is"

Edit: There's also a ">" left on the end too

Reply:
Will fix tonight.

Reply:
Typo fixed and I added above as comment. I want to keep `Scenario Outline` name short and simple as it's shown in allure report so unified description that will fit others future conflict is better.

Reply:
Fixed